### DISCUSSION TITLE
cron of strelka filecheck keeps generating a mail once a minute.

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#9314] [Unsupported Versions] [answered]
Hi,

After updating to 2.3.190, cron of strelka filecheck keeps generating a mail once a minute.
> **Title:**
> `Cron <socore@HOSTNAME> ps -ef | grep filecheck | grep -v grep || python3 /opt/so/conf/strelka/filecheck >> /opt/so/log/strelka/filecheck_stdout.log 2>&1 &`
> **Body:**
> `socore    6959  6924  0 Dec06 ?        00:00:00 python3 /opt/so/conf/strelka/filecheck`
> 
At least, STDOUT of first half command (i.e. `ps -ef | grep filecheck | grep -v grep`) should be pruned.
Following command is just a suggestion for the cron:
`ps -ef | grep filecheck | grep -v grep 1> /dev/null || python3 /opt/so/conf/strelka/filecheck >> /opt/so/log/strelka/filecheck_stdout.log 2>&1 &`

Cheers

**Comment by dougburks** [comment, marked as answer]
Please try the hotfix we just released:
https://blog.securityonion.net/2022/12/security-onion-23190-hotfix-20221207.html
